ui/lib/pki/addon/routes/keys/create.js

```diff
@@ -1,3 +1,18 @@
-import Route from '@ember/routing/route';
+import PkiKeysIndexRoute from '.';
+import { inject as service } from '@ember/service';
+import { withConfirmLeave } from 'core/decorators/confirm-leave';
 
-export default class PkiKeysCreateRoute extends Route {}
+@withConfirmLeave()
+export default class PkiKeysCreateRoute extends PkiKeysIndexRoute {
+  @service store;
+  @service secretMountPath;
+
+  model() {
+    return this.store.createRecord('pki/key');
+  }
+
+  setupController(controller, resolvedModel) {
+    super.setupController(controller, resolvedModel);
+    controller.breadcrumbs.push({ label: 'generate' });
```

Comment:
Should this be `create`?

Reply:
Since key's are generated, I thought we opted for using 'generate' even though we're following CRUD and routing-wise this is all considered `create` 

we could also change the routing if this ended up not being as intuitive as we thought 🤔